src/main/java/screens/DeleteListingsScreen.java

```diff
@@ -0,0 +1,85 @@
+package screens;
+
+import intefaceAdapters.deleteListing.DeleteListingController;
+import intefaceAdapters.deleteListing.DeleteListingResponseFormatter;
+import intefaceAdapters.deleteListing.DeleteListingScreenInterface;
+import useCases.deleteListing.DeleteListingDsRequestModel;
+import useCases.deleteListing.DeleteListingPresenter;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.List;
+
+public class DeleteListingsScreen extends JPanel implements DeleteListingScreenInterface, ActionListener {
+    private final DeleteListingController controller;
+    private final JPanel contentPanel;
+    private final JButton exit;
+    private final ListingCRUDScreen parent;
+
+
+    public DeleteListingsScreen(ListingCRUDScreen parent) {
```

Comment:
Good idea passing the frame as a parent so we can easily call change the panel in the frame!

Reply:
thank u